docs/guides/lustre-tuning.md

```diff
@@ -0,0 +1,32 @@
+# Lustre Tuning
+`/capstor/` and `/iopsstor` are both [lustre](https://lustre.org) filesystem.
+Lustre is an open-source, parallel file system used in HPC systems.
+As shown in ![Lustre architecture](/images/storage/lustre.png) uses *metadata* servers to store and query metadata which is basically what is shown by `ls`: directory structure, file permission, modification dates,..
+This data is globally synchronized, which means that handling many small files is not especially suited for lustre, and the perfomrance of that part is similar on both capstor and iopsstor.
```

Comment:
```suggestion
This data is globally synchronized, which means that handling many small files is not especially suited for Lustre, and the perfomrance of that part is similar on both capstor and iopsstor.
```
here and elsewhere.

Reply:
```suggestion
This data is globally synchronized, which means that handling many small files is not well suited for lustre.
```
Leave out the second part? Iopsstor should handle small files slightly better than Capstor, no? I may very well also be mistaken in which case it's obviously good to leave it in in.

Reply:
Some of this motivation overlaps with https://docs.cscs.ch/guides/storage/#many-small-files-vs-hpc-file-systems.

Reply:
The performance of the metadata part *is* roughly the same on iopsstor and capstor, it depends on the number of metadata servers and users adding load to it. For a while backup was putting extra load on capstor. Datawriting of many small files might indeed be slightly better on iopsstor because SSD are faster